### PR TITLE
Added DLT_FIRA_UCI and LINKTYPE_FIRA_UCI

### PR DESCRIPTION
**Pull request by hchataing**

### CODE REVIEW

File: pcap-common.c [hunk 1]
```diff
@@ -1228,7 +1228,14 @@
  */
 #define LINKTYPE_ZBOSS_NCP	292
 
-#define LINKTYPE_MATCHING_MAX	292		/* highest value in the "matching" range */
+/*
+ * Ultra-wideband (UWB) controller interface protocol (UCI).
+ * Protocol description on the FiRa consortium website:
+ *  https://groups.firaconsortium.org/wg/members/document/1679
```
Comment (mcr) [inline pcap-common.c:1234]: This links to a members only document.  Is there a document that is public?

Comment (hchataing) [reply pcap-common.c:1234]: Sadly none that I am aware of...

Comment (guyharris) [reply pcap-common.c]: So do you have a publicly-available description of the protocol?

Comment (hchataing) [reply pcap-common.c]: The FiRa has not made the specification public. 
Please check out the discussion on https://github.com/the-tcpdump-group/tcpdump-htdocs/pull/28; 
I added the packet framing description; still waiting on review for this.

Comment (hchataing) [reply pcap-common.c]: The link type 299 was approved on tcpdump htdocs. @guyharris Can you pickup the review for this MR ? Thanks !

**Comment by infrastation** [comment]
For posterity, `LINKTYPE_FIRA_UCI` became available in libpcap 1.10.6.

**Pull request merged**